src/components/basic/input/Select.tsx

```diff
@@ -48,7 +49,7 @@ const Options = styled.select`
   outline: none;
   font-size: inherit;
   background-color: transparent;
-  color: inherit;
```

Comment:
The background of the dropdown on my end was black so this worked... But coloring to black will make the text hard to see on blue background.